web-app/prisma/schema.prisma

```diff
@@ -719,6 +722,49 @@ model JobShare {
   @@map("share")
 }
 
+model JobSchedule {
+  id               String   @id @default(cuid())
+  createdAt        DateTime @default(now())
+  updatedAt        DateTime @updatedAt
+
+  // Ownership
+  userId           String
+  user             User     @relation(fields: [userId], references: [id], onDelete: Cascade)
+  organizationId   String?
+  organization     Organization? @relation(fields: [organizationId], references: [id], onDelete: SetNull)
+  agentId          String
+  agent            Agent    @relation(fields: [agentId], references: [id])
+
+  // Execution configuration
+  scheduleType     ScheduleType
+  cron             String?
+  oneTimeAtUtc     DateTime?
+  timezone         String   // IANA timezone identifier used for display and next run calculation
+  endOnUtc         DateTime?
+  endAfterOccurrences Int?
+  occurrenceCount  Int      @default(0)
+
+  // Job payload snapshot
+  inputSchema      Json
+  input            String
+  maxAcceptedCents BigInt
+
+  // State
+  isActive         Boolean  @default(true)
+  lastRunAt        DateTime?
```

Comment:
If I'm not mistaken we do not need this value. Do we? If we yes for what. I couldn't find a reason in the code for this information. Related to https://github.com/masumi-network/sokosumi/pull/1646/files#r2447569623

Reply:
We now have the latest job attached, and the creation date of this job is `lastRunAt`, correct? We have two sources of truth, and we need to keep `lastRunAt` in sync. 🤔 

We should be able to use Prisma to query the data for `lastRunAt` from the jobs table without needing to save it in the database. If I'm not mistaken, Prisma provides calculation queries out of the box.

I highly recommend maintaining a single source of truth instead of adding another one, as this would create additional overhead by requiring us to ensure the field remains in sync with the latest job.